src/upgradeConfig.js

```diff
@@ -2,7 +2,11 @@ const { presets: oldPresets, plugins: oldPlugins } = require('./packageData');
 
 // TODO: fix all of this
 function changePresets(config) {
-  const presets = config.presets;
+  let presets = config.presets;
+
+  if (!Array.isArray(presets) && typeof presets === 'string') {
+    presets = config.presets = [config.presets];
```

Comment:
We actually need it to be an array of strings not just make it an array 😛 
So like a split on ","?

Reply:
Oops, I'll fix